.gitignore

```diff
@@ -0,0 +1,34 @@
+# Created by https://www.toptal.com/developers/gitignore/api/java,maven
+# Edit at https://www.toptal.com/developers/gitignore?templates=java,maven
+
+### Java ###
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
```

Comment:
Please remove the extensions are not in use.

Reply:
Removed some lines and added Maven's `target/`

Reply:
It might be useful to keep .log files excluded from the repository (in case you will use any logging system in the project)